Create ui-c8y-1023-14-25-alarm-or-event-label-is-now-displayed-in-legend-instead-of.md (#4067)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-14-25-alarm-or-event-label-is-now-displayed-in-legend-instead-of.md

---
date: ""

> title: Alarm or event label is now displayed in legend instead of type.

Reviewer: ```suggestion
title: Data point graph legend now displays alarm/event labels instead of the type
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65446
version: 1023.14.25
---

> # Backport

Reviewer: ```suggestion
Previously, the legend for alarms and events in the "Data point graph" widget displayed the type of the entries. This issue has been resolved. The legend now correctly displays the configured alarm or event label instead.
```
